README.md

```diff
@@ -259,9 +234,7 @@ To follow these steps, you will need to have Node.js version 14.15 or greater in
 
         docker compose logs -f nginx
 
-3. When started initially, nginx would not have certificates at all (`$njs_acme_dir`, e.g. `/etc/nginx/njs-acme/`), so we can issue a new one by sending an HTTP request to a location with the `js_content` handler:
-
-        curl -vik --resolve proxy.nginx.com:8000:127.0.0.1 http://proxy.nginx.com:8000/acme/auto
+3. When started initially, nginx will not have certificates at all. If you use the [example config](examples/nginx.conf), you will need to wait one minute for the `js_periodic` directive to invoke `acme.clientAutoMode` to create the certificate.
```

Comment:
Something I don't understand: Can we have the ability to kickstart via a request as well as having the periodic process run?  It seems like perhaps the `js_periodic` invocation could be in a normal location block that kicks it off and we could have both?  Very possible that there's a situation I'm missing here but this change seems to take away some flexibility. 

Should we add documentation here to show how to do it the old way?
```javascript
js_content acme.clientAutoModeHTTP;
```
It could be that some users could prefer to invoke the check using an outside source as should in the deleted documentation.

Reply:
Sweet. I've documented it as an option in the `README.md`. LMK what you think...